hil/cli/node.py

```diff
@@ -0,0 +1,182 @@
+"""Commands related to node are in this module"""
+import click
+import sys
+from hil.cli.client_setup import setup_http_client
+
+C = None
+
+
+@click.group()
+def node():
+    """Commands related to node"""
+    global C
+    C = setup_http_client()
+
+
+@node.command(name='list')
+@click.argument('free', type=click.Choice(['free', 'all']), required=True)
+def nodes_list(free):
+    """List all nodes or free nodes"""
+    q = C.node.list(free)
+    if free == 'all':
+        sys.stdout.write('All nodes %s\t:    %s\n' % (len(q), " ".join(q)))
+    else:
+        sys.stdout.write('Free nodes %s\t:   %s\n' % (len(q), " ".join(q)))
+
+
+@node.command(name='show')
+@click.argument('node')
+def node_show(node):
+    """Show node information"""
+    q = C.node.show(node)
```

Comment:
q here works fine since it only exists within this scope. idk. just something thoughts